include/CL/cl_exp_defined_builtin_kernels.h

```diff
@@ -90,7 +90,9 @@ typedef enum
   POCL_CDBI_GAUSSIAN3X3_P512 = 37,
   POCL_CDBI_DBK_EXP_GEMM = 38,
   POCL_CDBI_DBK_EXP_MATMUL = 39,
+  /* see portablecl.org for details on usage */
```

Comment:
Please add a user manual reference instead of a web site domain.

Reply:
changed ref to section name and file where it is described.